packages/neuron-wallet/src/database/chain/ormconfig.ts

```diff
@@ -133,28 +144,54 @@ const connectOptions = async (genesisBlockHash: string): Promise<SqliteConnectio
       IndexerTxHashCacheRemoveField1701234043431,
       CreateCellLocalInfo1701234043432,
     ],
+    subscribers: [
+      AddressSubscribe,
+      AssetAccountSubscribe,
+      MultisigConfigSubscribe,
+      SudtTokenInfoSubscribe,
+      TxDescriptionSubscribe,
+    ],
     logger: 'simple-console',
     logging,
     maxQueryExecutionTime: 30,
   }
 }
 
-export const initConnection = async (genesisBlockHash: string) => {
+const initConnectionWithType = async (genesisBlockHash: string, connectionName: ConnectionName) => {
   // try to close connection, if not exist, will throw ConnectionNotFoundError when call getConnection()
   try {
-    await getConnection().close()
+    await getConnection(connectionName).close()
   } catch (err) {
     // do nothing
   }
-  const connectionOptions = await connectOptions(genesisBlockHash)
+  const connectionOptions = await connectOptions(genesisBlockHash, connectionName)
 
   try {
     await createConnection(connectionOptions)
-    await getConnection().manager.query(`PRAGMA busy_timeout = 3000;`)
-    await getConnection().manager.query(`PRAGMA temp_store = MEMORY;`)
+    await getConnection(connectionName).manager.query(`PRAGMA busy_timeout = 3000;`)
+    await getConnection(connectionName).manager.query(`PRAGMA temp_store = MEMORY;`)
   } catch (err) {
     logger.error(err.message)
   }
 }
 
+export async function initConnection(genesisBlockHash: string) {
+  await initConnectionWithType(genesisBlockHash, 'full')
+  await initConnectionWithType(genesisBlockHash, 'light')
+}
+
+export function migrateDBFile(genesisBlockHash: string) {
```

Comment:
Will the original DB file be removed?

Reply:
Currently, it will not be removed. Then we can easily revert the migration and start testing again.

Reply:
Does that mean the old db is intended for recovery during testing, and will be removed in production?
If so, I would suggest removing the old one at first but leaving a message about how to backup it. By doing so, the migration process can be fully tested, including removing old db.